pubspec.yaml

```
@@ -14,5 +14,5 @@ flutter:
         - asset: lib/fonts/materialdesignicons-webfont.ttf
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
+  sdk: ">=2.12.0 <3.1.0"
```

Should support all 3.x versions:
```suggestion
  sdk: ">=2.12.0 <4.0.0"
```